exception_handling_task_2.js

```diff
@@ -0,0 +1,17 @@
+// When running 1/0, the error 'cannot be divided by zero'
+
+function divide(a, b) {
```

Comment:
This is not the right solution, you will handle the error in an if else logic block, not with a try catch. In javascript the result of division by 0 is Infinity, you need to use this value and throw an error

Reply:
function divide(a, b) {
    try {
        if(a / b === Infinity) {
            throw new Error;
        } else {
            return a / b;
        }
    } catch (err) {
        console.log('cannot be divided by zero')
    }
}